### ISSUE TITLE
[A11Y] Search results tabs are inaccessible

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5964]
**Describe the bug**

The search result type tabs are not accessible using keyboard, and their role in relation to the result is unclear.

**To Reproduce**

1. Go to category display page
2. Enter a search keyword and confirm
3. On the search page, try to access the tabs using a keyboard
4. Observe that the tabs are not focusable

**Expected behavior**

Tabs are accessible using the keyboard, and their role is correctly announced as "tab" by the screen readers.

**Additional context**

- https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
- https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html
